scripts/airtable-auto.py

```diff
@@ -0,0 +1,47 @@
+# copied directly from Colby's earlier work:
+# https://github.com/covid19risk/legacy_website/pull/72/files
+
+from airtable import Airtable
+import json
+import os
+
+def valid_entry(entry):
+    needed_values = ['type', 'author_name', 'outlet_name', 'title', 'date']
+
+    for value in needed_values:
+        if value not in entry:
+            return False
+
+    return True
+
+
+def main():
+    try:
+        # these keys have been set up Github /websites repo secrets already
+        airtable = Airtable(os.environ['AIRTABLE_BASE_KEY'], 'news', api_key=os.environ['AIRTABLE_API_KEY']) 
+        
+        # hardcoding base key and api key for now:
+        # airtable = Airtable(os.environ['app2gjpkAcqIzagxn'], 'news', api_key=os.environ['keyr0gMB3vBqrwhf0']) 
```

Comment:
hmmm

Reply:
Whoops, I thought I had gotten rid of that line!  I reset my API key now and we can potentially use a different Base (means a different Airtable table basically) to avoid any potential risk of bad actors.